tests/Microsoft.Identity.Web.Test/Microsoft.Identity.Web.Test.csproj

```diff
@@ -8,8 +8,12 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.0.1" />
-    <PackageReference Include="xunit" Version="2.4.0" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.0" />
+    <PackageReference Include="NSubstitute" Version="4.2.1" />
```

Comment:
><PackageReference Include="NSubstitute" Version="4.2.1" /> [](start = 4, length = 58)

Did you consider if we can make code changes instead? (I know we use it in MSAL .NET and it's serving the purpose well).

Reply:
Yes. Keeping track of changes.